### PR TITLE
#7 feat: Reusable Card Component

### PR DESCRIPTION
**Pull request by KeithWesley254**
Made a reusable card component and displayed it on the home page in a flex and added more variable styles to the main scss.

Screenshot 2023-02-04 at 00 13 44

Screenshot 2023-02-04 at 00 13 31

### CODE REVIEW

File: apps/elewa-group-website/src/styles.scss [hunk 1]
```diff
@@ -1 +1,12 @@
...
+    --card-bgColor: #f6f6f6;
+    --card-text-color: #000000;
+    --card-font-size: 17px;
+    --card-header-font-size: 24px;
+}
+
+*{
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/styles.scss:12]: Move this styles to _variables.scss file and use domain naming i.e --elewa-group-website-card-bgColor

File: libs/features/components/cards/src/lib/elewa-group-card/elewa-group-card.component.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  
+  h3,
+  h5 {
+    font-weight: 400;
+  }
+  
+  h3 {
+    font-size: var(--card-header-font-size);
+  }
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/elewa-group-card/elewa-group-card.component.scss:32]: Put variables in _variables.scss file

File: libs/features/components/cards/src/lib/elewa-group-card/elewa-group-card.component.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.card {
+  width: 320px;
+  height: 350px;
+  background-color: var(--card-bgColor);
+  color: var(--card-text-color);
+  border-radius: 20px;
+  overflow: hidden;
+  margin: 10px;
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/elewa-group-card/elewa-group-card.component.scss:8]: Use font family variable provided in _variables.scss

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './pages/home-page/home-page.component';
+import { CardsModule } from '@elewa-group/features/components/cards';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:5]: Order of imports - Local imports come last i.e

import { HomePageComponent } from './pages/home-page/home-page.component'; should be last import

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by KeithWesley254** [comment]
@IanOdhiambo9 I made the changes

**Pull request closed without merging**